src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/__tests__/ExposureSubmissionUITests.swift

```diff
@@ -112,55 +112,178 @@ class ENAUITests_04_ExposureSubmissionUITests: XCTestCase {
 		// Accept the alert.
 		XCTAssertTrue(app.alerts.firstMatch.exists)
 		app.alerts.buttons.firstMatch.tap()
-
 	}
 
-	func test_SubmitTAN() {
-		// Setup service mocks.
-		app.launchArguments += [UITestingParameters.ExposureSubmission.useMock.rawValue]
-		app.launchArguments += [UITestingParameters.ExposureSubmission.getRegistrationTokenSuccess.rawValue]
-		app.launchArguments += [UITestingParameters.ExposureSubmission.submitExposureSuccess.rawValue]
-		launch()
+	func test_SubmitTAN_SymptomsOptionNo() {
```

Comment:
Thank you for adding UI tests that cover your new screens! 👍

Reply:
☺️